packages/nl-design-system-unstable/__docs__/form-patterns.md

```diff
@@ -1,139 +1,270 @@
+Formulierelementen moeten aan de volgende eisen voldoen, ze hebben:
 
-## Accessibility
+* Een label of instructie die is gekoppeld aan het formulierelement,
+* Bij een fout een duidelijke foutmelding,
+* Bij een foutmelding moet duidelijk worden uitgelegd hoe deze fout kan worden verholpen en hoe deze in de toekomst kan worden voorkomen.
+* Als een invoer een fout genereert, zorg er dan voor dat de bijbehorende foutmelding op het juiste moment en op de juiste manier door voorleessoftware wordt uitgesproken.
 
-Form elements must meet the following requirements, they have:
+## Formulier interactie; vragen en antwoorden
 
-1. A label or instruction associated with the form element,
-2. In case of an error, a clear error message,
-3. In the event of an error message, a clear explanation must be given on how to correct this error and how to prevent it in the future.
+Vraag alleen om de strikt noodzakelijke informatie en informeer gebruikers over de noodzaak van de gevraagde informatie.
 
-If an input generates an error, make sure that the corresponding error message is announced by screen readers at the right time and in the right way.
+Bij iedere vraag dien je je af te vragen of deze daadwerkelijk nodig is voor de bezoeker en/of organisatie. Is informatie al binnen de organisatie aanwezig, vraag je dan af of deze nogmaals gevraagd dient te worden, bv. ter controle of ter wijziging door de bezoeker.
+
+Indien het onduidelijk is waarom informatie gevraagd wordt informeer bezoekers dan over de noodzaak hiervan maar ook wat de voordelen voor hen zijn.
+
+Voorkom dat gebruikers dezelfde informatie meerdere keren moeten invoeren.
+
+### Tekst labels
+
+Labels zijn de vragen die je een gebruiker stelt. Hier ga je een dialoog aan, dus zorg dat deze zo duidelijk en vriendelijk maar tevens beknopt mogelijk zijn. Laat een bezoeker de vragen makkelijk scannen.
+
+Een goed label wordt buiten een invoerveld geplaatst, niet daarbinnen (de zg. _‘floating labels’_). Op deze manier heeft het label altijd een vaste positie en kan het in een leesbare grootte worden weergegeven. Een label dat in een invoerveld is geplaatst, kan een bezoeker in verwarring brengen, aangezien het voor reeds ingevoerde tekst aangezien kan worden. De leesbaarheid lijdt er ook onder omdat het contrast tussen tekst voorgrond- en achtergrondkleur vaak te laag is. Het kan ook verward worden met inactieve of placeholder tekst. Bovendien kan het ook nodig zijn om soms daadwerkelijke placeholder tekst te tonen.
+
+Een bijkomend voordeel van het plaatsen van een label buiten het invoerveld is dat het klik- en tap-oppervlakte automatisch groter is, waardoor het selecteren van een invoerveld voor zowel muis- als touchscreen gebruikers makkelijker is.
+
+#### Positie van labels
+
+Overweeg om labels links boven een invoerveld te plaatsen. Dit biedt meerdere voordelen die bijdragen aan de gebruiksvriendelijkheid.
```

Comment:
Misschien kunnen we hier al localisatie voor inbedden? Dus niet "links" gebruiken, maar tekst-start boven een invoerveld o.i.d.? @Robbert heb jij inspiratie voor een meer gelokaliseerde verwoording die wel lekker loopt?

Reply:
"op de regel boven het invoerveld"?